### DISCUSSION TITLE
Unifying CIECAM02 & CAM16.

### DISCUSSION THREAD

**Discussion by TFiFiE** [colour-science/colour#719] [Ideas]
Would it be in the scope of this project to consolidate as much as CIECAM02 and CAM16 have in common in a single codebase and make explicit exactly how they differ?

**Comment by thomasmansencal** [comment]
Hi @TFiFiE,

From a code standpoint, CAM16 almost entirely piggyback on CIECAM02, if you look at the module imports [it is quite significant](https://github.com/colour-science/colour/blob/develop/colour/appearance/cam16.py#L34).  I prefer the separation of concerns, makes the code a bit more manageable and importantly, CAM16 is not CIECAM16, it is not blessed by the CIE, so I would rather keep it separate. Hope that makes sense.

Cheers,

Thomas
